main.go

```diff
@@ -7,6 +7,8 @@ import (
 
 	_ "terraform-provider-genesyscloud/genesyscloud/tfexporter"
 
+	_ "github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud"
```

Comment:
I would add a comment to not remove this line and the reason why.

Reply:
Will do. 

I should mention as well, I just tried changing the package name in `apidocs/main.go` from `main` to `apidocs` so that it could be imported into `{root}/main.go` and `{root}/genesyscloud/provider.go` but no luck with that approach either.